beta/src/pages/learn/conditional-rendering.md

```diff
@@ -52,9 +52,9 @@ export default function PackingList() {
 
 </Sandpack>
 
-Notice that some of the `Item` components have their `isPacked` prop set to `true` instead of `false`. You want to add a checkmark (✔) to packed items if `isPacked={true}`.
+ध्यान दें कि कुछ `Item` कौम्पोनॅन्ट का `isPacked` प्रोप `false` के बजाय `true` पर सेट है। आप पैक किए गए आइटम में एक चेकमार्क (✔) जोड़ना चाहते हैं यदि `isPacked={true}` है।
```

Comment:
`प्रोप` => `prop`

we are not translating core React terminologies

Reply:
okay.